CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 # UNRELEASED
 
+# 0.24.0
```

Comment:
```suggestion
# 0.23.0
```
Let's just make this next release the 0.23.0 release after the pocket IC changes